test/cypress/support/resource_commons.ts

```diff
@@ -22,3 +21,7 @@ export function waitUntilEmpty(resource, search, numberOfRetriesLeft = 60, waiti
         waitUntilEmpty(resource, search, numberOfRetriesLeft - 1, waitingInterval);
     });
 }
+
+export function minutesToMs(minutes) {
```

Comment:
Cool idea to extract this function :+1: 

Minor:

We could add a type annotation :smile: 

```suggestion
export function minutesToMs(minutes: number) {
```

Reply:
Right 👍🏻 As the file is `.ts` now, that's good idea to add it.